patches/server/0004-Test-changes.patch

```diff
@@ -52,7 +52,7 @@ index 0000000000000000000000000000000000000000..286790061c9d81c872108ef63e1a1aba
 +            return;
 +        }
 +
-+        final Server dummyServer = mock(Server.class);
++        final Server dummyServer = mock(Server.class, org.mockito.Mockito.withSettings().stubOnly());
```

Comment:
Import it here, this is a new file Paper made.

Reply:
Done :+1: